_layouts/loop.html

```diff
@@ -234,6 +234,22 @@ <h2><a href="{{ site.url | append: site.baseurl | append: post.url }}" title="Re
 				{% endif %}
 			{% endfor %}
 			</ul>
+
+		
+		{% elsif page.collection == "topics" %}
+				
+			<ul>
+			{% for post in site.topics reversed %}
```

Comment:
Could we use the variable 'topic' instead of 'post' here? Makes it clear what we are referring to
```suggestion
			{% for topic in site.topics reversed %}
```